deploy/complete/docker-compose/docker-compose.yml

```diff
@@ -135,6 +135,7 @@ services:
     image: iad.ocir.io/oracle/ateam/mushop-orders:2.1.1
     ports:
       - 8085:80
+      # - 9091:9091
```

Comment:
Remove comments if not using/unnecessary.

Reply:
done